schema.json

```diff
@@ -93,6 +93,10 @@
                 "isResponseDictionary": {
                     "type": "boolean",
                     "description": "(Optional) Set to true if the batch resource returns the results as a dictionary with key mapped to values (instead of a list of items). If this option is supplied `reorderResultsByKey` should not be. Default: false"
+                },
+                "isBatchKeyASet": {
+                    "type": "boolean",
+                    "description": "(Optional) Set to true if the interface of the resource takes the batch key as a set (rather than an array). Default: false"
```

Comment:
Might help to also note in the description what swagger flag can cause this to need to be True, so maybe a 
`when uniqueItems: true in the swagger spec`